mitodl/open-discussions
# Add spinner for Load more comments link

#### What are the relevant tickets?
Fixes #369 

#### What's this PR do?
Adds a spinner which shows up while more comments are loading

#### How should this be manually tested?
Click multiple 'Load more comments' buttons concurrently. While they are spinning click them again to make sure they are disabled
